src/js/stores/DCOSStore.js

```diff
@@ -98,6 +113,21 @@ class DCOSStore extends EventEmitter {
     this.emit(DCOS_CHANGE);
   }
 
+  onMarathonQueueChange(nextQueue) {
+    let {marathon:{queue}} = this.data;
+
+    nextQueue.forEach((entry) => {
+      if (entry.app == null) {
+        return;
+      }
+
+      let serviceId = entry.app.id;
+      queue.set(serviceId, entry);
```

Comment:
I'm ok with that and would even encourage us to do this more often, but I think we agreed that we not store properties (for what ever reason) if we only access it once. Your call.